src/sidebar/store/modules/annotations.js

```diff
@@ -254,15 +254,15 @@ function addAnnotations(annotations, now) {
 
   return function(dispatch, getState) {
     const added = annotations.filter(function(annot) {
-      return !findByID(getState().annotations, annot.id);
+      return !findByID(getState().base.annotations, annot.id);
```

Comment:
thunk actions are not scoped so they need to drill into the "base" namespace until the module is converted.